config/example/game_options.txt

```diff
@@ -9,6 +9,9 @@ HEALTH_THRESHOLD_CRIT -50
 ## level of health at which a mob becomes dead
 HEALTH_THRESHOLD_DEAD -100
 
+## Comment this line to prevent humans showing a visible message upon death ('X seizes up then falls limp, eyes dead and lifeless').
+SHOW_HUMAN_DEATH_MESSAGE

```

Comment:
IMO should start commented out, to keep current behavior

Reply:
Fair call.